tests/integration/features/callapi/public.feature

```diff
@@ -5,105 +5,105 @@ Feature: callapi/public
-    And user "guest" sees 0 peers in call "room" with 404
-    Then user "participant1" joins room "room" with 200
-    Then user "participant1" sees 0 peers in call "room" with 200
-    And user "participant2" sees 0 peers in call "room" with 200
-    Then user "participant1" joins call "room" with 200
-    Then user "participant1" sees 1 peers in call "room" with 200
-    And user "guest" sees 0 peers in call "room" with 404
-    Then user "guest" joins room "room" with 200
-    Then user "participant1" sees 1 peers in call "room" with 200
-    And user "guest" sees 1 peers in call "room" with 200
-    And user "guest" joins call "room" with 200
-    Then user "participant1" sees 2 peers in call "room" with 200
-    And user "guest" sees 2 peers in call "room" with 200
-    Then user "guest" leaves call "room" with 200
-    Then user "participant1" sees 1 peers in call "room" with 200
-    And user "guest" sees 1 peers in call "room" with 200
-    Then user "guest" leaves room "room" with 200
-    Then user "participant1" sees 1 peers in call "room" with 200
-    And user "guest" sees 0 peers in call "room" with 404
-    Then user "participant1" leaves call "room" with 200
-    Then user "participant1" sees 0 peers in call "room" with 200
-    And user "guest" sees 0 peers in call "room" with 404
-    Then user "participant1" leaves room "room" with 200
-    Then user "participant1" sees 0 peers in call "room" with 200
-    And user "guest" sees 0 peers in call "room" with 404
+    And user "participant1" adds user "participant2" to room "room" with 200 (v4)
+    Then user "participant1" is participant of room "room" (v4)
+    And user "guest" sees 0 peers in call "room" with 404 (v4)
+    Then user "participant1" joins room "room" with 200 (v4)
+    Then user "participant1" sees 0 peers in call "room" with 200 (v4)
+    And user "participant2" sees 0 peers in call "room" with 200 (v4)
+    Then user "participant1" joins call "room" with 200 (v4)
+    Then user "participant1" sees 1 peers in call "room" with 200 (v4)
+    And user "guest" sees 0 peers in call "room" with 404 (v4)
+    Then user "guest" joins room "room" with 200 (v4)
+    Then user "participant1" sees 1 peers in call "room" with 200 (v4)
+    And user "guest" sees 1 peers in call "room" with 200 (v4)
+    And user "guest" joins call "room" with 200 (v4)
+    Then user "participant1" sees 2 peers in call "room" with 200 (v4)
+    And user "guest" sees 2 peers in call "room" with 200 (v4)
+    Then user "guest" leaves call "room" with 200 (v4)
+    Then user "participant1" sees 1 peers in call "room" with 200 (v4)
+    And user "guest" sees 1 peers in call "room" with 200 (v4)
+    Then user "guest" leaves room "room" with 200 (v4)
+    Then user "participant1" sees 1 peers in call "room" with 200 (v4)
+    And user "guest" sees 0 peers in call "room" with 404 (v4)
+    Then user "participant1" leaves call "room" with 200 (v4)
```

Comment:
after scrolling through pages and pages if sprinkled `(v4)` I wonder if we even need to mention it in the tests, considering that v4 is the new default and we're not going to test versions < v4 and only when v5 comes in, if incompatible, we would require to specify it

Reply:
So the thing is, the version was optional until now. I force it to now, because I noticed that all the tests used the v1 in 99% of the cases, so they didn't use/test the methods we use in the UI etc.
Sure we could go back to optional and just use v4 as fallback, but I'd rather be explicit making everyone aware that any other version is not tested.

Reply:
agreed.

when scrolling further I saw we still had some v1 steps, so making it explicit will make sure there is no confusion